realm/src/androidTest/java/io/realm/RealmTest.java

```diff
@@ -1800,13 +1800,14 @@ public void testMutableMethodsOutsideTransactions() throws JSONException, IOExce
     // TODO: re-introduce this test mocking the ReferenceQueue instead of relying on the GC
```

Comment:
Can we reintroduce the test? I mean, you are making changes to a test which is not used.